scripts/helpers/moduleNames.js

```diff
@@ -0,0 +1,8 @@
+// moduleNames.js
```

Comment:
Let's remove this comment, add the copyright template, and maybe rename this file to something like `excludableModules`